lib/perl/Genome/VariantReporting/Framework/Plan/ExpertPlan.pm

```diff
@@ -21,21 +21,22 @@ sub category {
 
 sub needed_translations {
     my $self = shift;
-    return Set::Scalar->new(map {$self->adaptor_params->{$_}}
-        $self->get_class->adaptor_class->required_translated_input_names);
+    my $needed = Set::Scalar->new(map {$self->run_params->{$_}} $self->get_class->run_class->required_translated_input_names);
+    $needed->insert(map {@{$self->run_params->{$_}}} $self->get_class->run_class->required_translated_is_many_input_names);
```

Comment:
Since we had to fix this both here and in the plan base, what if we would fix required_translated_input_names to include both is_many and non-is_many properties?